instrumentation/instagocb/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/instana/go-sensor/instrumentation/instagocb
+
+go 1.13
```

Comment:
@sanojsubran: It looks like they released a new version two days ago. 😞 I initially started this instrumentation to support version 2.6.5, and now they've come out with 2.7.0.

The Go version has been updated from 1.13 to 1.19 in this major release.

In my opinion, we can stick with Go version 1.13, as this is a significant new release from Couchbase, and many users will likely continue using the 2.6 version for an extended period.

@instana/eng-go, what are your thoughts on this?

Reply:
Oh, Then it is absolutely fine. I was just trying to say that we can keep it consistent with the original package. I was also wondering how the tests went through if the min version is go1.13. Usually something will break since they will be using newer features of Go.